### ISSUE TITLE
[Feature Request] Battle Mode (Toggles elements on and off when weapon drawn or in combat)

### ISSUE THREAD

**Issue by Rellawing** [Caraxi/SimpleTweaksPlugin#440]
tl:dr;

Without the Fader plugin, I've fallen back on using my old hudlayout macros to try to compensate for what's now missing for the sake of a clean UI under certain circumstances.

While this old-school method is adequate and with the new chat hiding tweaks, we have close to the same level of functionality, but with a few niggling issues leftover.

This proposed tweak would make the macros unnecessary (although we'd be sol whenever plugins are knocked out by updates) and more intelligently handle what trays show given criteria/conditions.

More powerful additional option: Different rules for different HUD Layouts for those who role-play, emote, play music, or whatever. But that would require a trigger for while performing, or whatever else.

Some of this can be done using QoL Bar, but I wouldn't relish making QoL Bars for every single job, spending much of a day or half of it at least only to have to laboriously tweak them every time Square adjusts abilities and rotations.

I miss the focus key held feature that Fader had; ctrl, alt, shift, etc to toggle trays on and off as needed/wanted. I would additionally request this feature included.

Imagine having three or so hotbars overlapped perfectly positioned over each other, showing and hiding based on which modifier is currently held.

This is a dream for clean UI fans like me.

Thanks for your consideration.

**Comment by Diorik** [comment]
Fader has since come back, and this also can be accomplished with HudManager. Seems way outside the scope of simple tweaks.

**Comment by Rellawing** [comment]
Fader handles it! Excellent! I've never heard of this HudManager though... is it a plugin outside Dalamud's regular lineup?
